langchain/retrievers/azure_cognitive_search.py

```diff
@@ -52,10 +53,25 @@ def validate_environment(cls, values: Dict) -> Dict:
         )
         return values
 
-    def _build_search_url(self, query: str) -> str:
+
+    def _build_search_url(self, query: str, top_k: int) -> str:
```

Comment:
Cool that you added comments! I'm not sure though if we need them here because (i) it's an internal function (it starts with a `_`) and the function and variable names are pretty self-explanatory